zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchExporter.java

```diff
@@ -93,20 +93,20 @@ public void open(final Controller controller) {
 
   @Override
   public void close() {
+    if (client != null) {
```

Comment:
❓ I'm curious why are we introducing changes to `Elasticsearch Exporter`?  could you share some context?

Reply:
There is some dummy creation of exporter on startup in order to validate configuration IIRC. The instance is then closed, but throws NPE as it was not properly opened. I already fixed the same thing in Camunda exporter earlier, that's why I decided to clean up here also.